### PR TITLE
Youtube clone homepage

### PR DESCRIPTION
**Pull request by EricMbouwe**
Cloning a youtube page in HTML and CSS.
We create the structure of the page with the header, main and footer elements.
we add a video and its description in the first section of the main and the comments in the second section.
we create a sidebar for the recommended videos and their description.

### CODE REVIEW

**Comment by lekegitrepo** [comment]
### Hi team :+1: 
You have done a **great job** in this project but you still need to make some changes.
Make the following changes and submit for another code review.
1. As mentioned in the [pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)
icons_requirement
2. [optional] Try to add little details to your pull request.
3. Remove .vscode folder since it's irrelevant [pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
gitignore
4. Fix these errors

image

**[Optional]** This Html semantic is not in right order. The basic structure for a page is:
```
<header></header>
<main></main>
<footer></footer>
```
instead of:
```
<header></header>
<div class="container">
  <main></main>
</div>
<footer></footer>
```

Make these changes and submit again for review.

Reviewed by: Temitayo Adeleke
-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

If you have any question you can always contact me on slack.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+            </form>
+        </div>
+
+        <div class="nav-bar-right-buttons">
+            <button type="submit"><i class="fas fa-upload"></i></button>
+            <button class="sign-in-button">Sign in</button>
+        </div>
+    </header>
+
+    <div class="container">
```
Comment (lekegitrepo) [inline index.html]: **[Optional]** This Html semantic is not in right order. The basic structure for a page is:
<header></header>
<main></main>
<footer></footer>

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
**Status: Request changes**

**Comment by davidmrtz-dev** [comment]
Pull request under review by David Elí [TSE]

**Comment by davidmrtz-dev** [comment]
### **Status: Approved.** ✔️ 
Hi Team. 😄 
Well done, you implemented all the changes required by the previous TSE. 💯 👍 
You can move onto the next project.

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

**Pull request merged**